src/context/mod.rs

```diff
@@ -226,6 +226,22 @@ pub trait Context<'a>: ContextInternal<'a> {
         result
     }
 
+    #[cfg(all(feature = "try-catch-api", feature = "napi-runtime"))]
```

Comment:
Note: For the final documented version, it would be beneficial to include the behavior that this method will `panic` if a user manually constructs an `Err(Throw)`.

Reply:
Yeah, when we un-feature-gate it, it'll need API docs. In the meantime I'll make sure that gets added to the RFC.

Reply:
I'll try to add an `Exceptions` section to the guides docs as well